modules/todosStatus.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] Hello, I would suggest that you should not disable linter because it is the tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Can you fix the issue without deleting the linter?
- [ ] **NB: Run your App to make sure it is working before submitting it for re-review**